src/api/app/views/webui/package/_submit_request_dialog.html.haml

```diff
@@ -1,57 +0,0 @@
-.modal.fade#submit-request-modal{ tabindex: -1, role: 'dialog', aria: { labelledby: 'submit-request-modal-label', hidden: true } }
-  .modal-dialog.modal-dialog-centered{ role: 'document' }
-    .modal-content
-      = form_tag({ controller: 'package', action: 'submit_request' }, method: 'post') do
-        .modal-header
-          %h5.modal-title#submit-request-modal-label Create Submit Request
-        .modal-body
-          %p
-            Do you want to submit #{package_link package, rev: revision}?
-          %p.font-italic
-            You are about to send a request to the owner of the original #{package ? 'package' : 'project'}. Please describe what you have done and
-            why they should apply your changes. The owner will be notified with the following details.
-
-            = hidden_field_tag(:project, project)
-            = hidden_field_tag(:package, package)
-            = hidden_field_tag(:rev, revision)
-            .form-group
-              = label_tag(:sourceproject, 'From source project:')
-              = text_field_tag(:sourceproject, project.name, disabled: true, class: 'form-control text-truncate')
-            .form-group
-              = render partial: 'webui/shared/autocomplete', locals: { html_id: 'targetproject', label: 'To target project:',
-                                                                disabled: params[:readonly], value: nil,
```

Comment:
Everything with `params[:readonly]` wasn't migrated to the new view. I couldn't see the reasoning behind this. Why would target project/package be disabled if their value is always set to `nil` when this view is rendered? It doesn't make sense. Submitting the form in this state simply fails.